src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -1350,16 +1379,16 @@ object LazyList extends SeqFactory[LazyList] {
 
   private object LazyBuilder {
```

Comment:
should we nuke this (or at least put it under a separate method)? there was some discussion a while ago that all builders should be eager. kind of unrelated to this PR I suppose